dataland-frontend/src/components/forms/parts/elements/basic/BaseDataPointFormField.vue

```diff
@@ -0,0 +1,86 @@
+<template>
+  <div class="form-field" :data-test="name">
+    <FormKit v-model="baseDataPoint" type="group" :name="name">
+      <slot />
+      <!-- TODO change this condition -->
```

Comment:
We are currently only using this component for YesNo and YesNoNa Radio Buttons as data point value provider. Thus I would leave the conditions using `"Yes"` and `"No"` as is. How would this behave if we have a base data point with some numeric value for example? Would it always be displayed in that case or only if a value is provided? 
If we leave it as is this is nothing we need to bother with now.
@SiegfriedSobkowiak